hosting_docs/source/backups_dr/2-backups-guide.rst

```diff
@@ -284,10 +284,28 @@ BlobDB backups create a compressed version of the blobdb data directory.
 
 Restoring BlobDB Backups
 ^^^^^^^^^^^^^^^^^^^^^^^^
-
-You can follow the same instructions as for `restoring couchdb <#restoring-couchdb-backups>`_ (extract the backup file into the blobdb data directory: ``/opt/data/blobdb/``\ ). 
-
-The files in the resulting directory should all be owned by the user ``cchq`` (i.e. you should be the ``cchq`` user when extracting the files)
+The BlobDB restore process depends on what BlobDB system you're using.
```

Comment:
Missing blank line, otherwise the bullets will be rendered as dashes inline.
```suggestion
The BlobDB restore process depends on what BlobDB system you're using.

```